tests/integration/conftest.py

```diff
@@ -39,13 +10,19 @@ def splinter_webdriver():
 
 
 @pytest.fixture(scope='session')
-def title():
-    return random_string()
+def title(request):
```

Comment:
I think you misunderstood.  This function is called "title" because it used to be a generic title, but now you've added code that assumes it's a brief title.